src/test/java/io/reactivex/flowable/FlowableTests.java

```diff
@@ -1125,6 +1126,34 @@ public Object apply(Flowable<Object> onSubscribe) {
         });
     }
 
+    @Test
+    public void testAsExtend() {
+        final TestSubscriber<Object> subscriber = new TestSubscriber<Object>();
+        final Object value = new Object();
+        Flowable.just(value).as(new FlowableConverter<Object, Object>() {
+            @Override
+            public Object apply(Flowable<Object> onSubscribe) {
+                    onSubscribe.subscribe(subscriber);
+                    subscriber.assertNoErrors();
+                    subscriber.assertComplete();
```

Comment:
Could not this run through without asserting if the Converter is never called?

Reply:
True... I could add something after this to assert that `subscriber` is complete?

Reply:
How about an AtomicBoolean that you set to true once it's called and you fail the test when it's set to false

Reply:
I think I got a better one - 5c3bdf3